components/raftstore/src/store/worker/refresh_config.rs

```diff
@@ -0,0 +1,214 @@
+// Copyright 2021 TiKV Project Authors. Licensed under Apache-2.0.
+use crate::store::fsm::apply::{ApplyFsm, ControlFsm};
+use crate::store::fsm::store::StoreFsm;
+use crate::store::fsm::PeerFsm;
+use batch_system::{BatchRouter, Fsm, FsmTypes, HandlerBuilder, Poller, PoolState, Priority};
+use file_system::{set_io_type, IOType};
+use std::fmt::{self, Display, Formatter};
+use std::sync::Arc;
+use std::thread;
+use tikv_util::worker::Runnable;
+use tikv_util::{debug, error, info, safe_panic, thd_name};
+
+pub struct PoolController<N: Fsm, C: Fsm, H: HandlerBuilder<N, C>> {
+    pub router: BatchRouter<N, C>,
+    pub state: PoolState<N, C, H>,
+}
+
+impl<N, C, H> PoolController<N, C, H>
+where
+    N: Fsm,
+    C: Fsm,
+    H: HandlerBuilder<N, C>,
+{
+    #[allow(dead_code)]
```

Comment:
Could we remove `#[allow(dead_code)]` of this file?

Reply:
For CI build, will remove at next PR